config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/users", to: "users#index", as: 'user'
+  get "/users/:id", to: "users#show", as: 'users'
+  get "/users/:user_id/posts", to: "posts#index", as: 'user_posts'
+  get "/users/:user_id/posts/:id", to: "posts#show", as: 'user_post'
```

Comment:
## Optional Change 
This change won't stop your project from being approved but will definitely help improve your code. 👍🏽
- [ ] What if you could shorten your routes like this:
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```